src/invoices/InvoiceDetails/VoucherInformation/VoucherInformation.js

```diff
@@ -0,0 +1,191 @@
+import React, { Fragment, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
```

Comment:
However, this empty line makes no sense, since `lodash` is the same 3rd-party lib.

Reply:
at least, we don't do this in Acquisitions-UI code